AsyncDisplayKit/Details/ASPINRemoteImageDownloader.m

```diff
@@ -53,23 +123,39 @@ - (void)fetchCachedImageWithURL:(NSURL *)URL
 
 - (void)clearFetchedImageFromCacheWithURL:(NSURL *)URL
 {
-  PINRemoteImageManager *manager = [PINRemoteImageManager sharedImageManager];
+  PINRemoteImageManager *manager = [self sharedPINRemoteImageManager];
   NSString *key = [manager cacheKeyForURL:URL processorKey:nil];
   [[[manager cache] memoryCache] removeObjectForKey:key];
 }
 
 - (nullable id)downloadImageWithURL:(NSURL *)URL
                       callbackQueue:(dispatch_queue_t)callbackQueue
-                   downloadProgress:(void (^)(CGFloat progress))downloadProgressBlock
-                         completion:(void (^)(UIImage *image, NSError * error, id downloadIdentifier))completion
+                   downloadProgress:(ASImageDownloaderProgress)downloadProgress
+                         completion:(ASImageDownloaderCompletion)completion;
 {
-  return [[PINRemoteImageManager sharedImageManager] downloadImageWithURL:URL options:PINRemoteImageManagerDownloadOptionsSkipDecode completion:^(PINRemoteImageManagerResult *result) {
+  return [[self sharedPINRemoteImageManager] downloadImageWithURL:URL options:PINRemoteImageManagerDownloadOptionsSkipDecode completion:^(PINRemoteImageManagerResult *result) {
```

Comment:
I might suggest making this simply [self imageManager], as this file requires PIRI and this method is called in many long lines